### ISSUE TITLE
On zoom can we fix the node size and label font,change only the edge length between the nodes?

### ISSUE THREAD

**Issue by SVRKishoreReddy** [almende/vis#3462]
Please make sure to **read the following list** before creating a new issue:

* This issue tracker is not supposed to be used for questions on how to use visjs. It is intended to be used for bug reports and feature requests! In case you face yourself with a usage question, then post your question e.g. on [stackoverflow](https://stackoverflow.com/questions/tagged/vis.js) tagged with "vis.js".
* Have you already used the [github search](https://github.com/almende/vis/issues), read the [documentation](http://visjs.org/) and looked at the [examples](https://github.com/almende/vis/tree/develop/examples)?
* Make sure to mention which vis-component (network, timeline, graph2D, graph3d) you are referring to.
* Make sure to use the [latest version of vis.js](https://cdnjs.com/libraries/vis) for bug reports.
* Make sure to mention which browser and OS you are using when creating a bug report.
* Please provide a minimal code example that demonstrates your issue. We recommend using [jsbin](jsbin.com) for that.
* Delete this list from the actual issue.

**Comment by SVRKishoreReddy** [comment]
It is like, if at a particular zoom level if two nodes are overlapping, then on zooming into those nodes,the gap between them should increase but the size and font of the node should remain the same.

**Comment by wimrijnders** [comment]
I'm not understanding the question very well; I'll translate it as such: 

> Is it possible to keep the size and font of nodes constant while zooming and out?

In regular usage, this is not possible. This is, however, the effect you get when you use HTML overlays. [Here's a demo](http://jsbin.com/behagul/1/edit?html,output), try zooming in and out. I don't know your requirements, but if you want to do styling with HTML, consider this option.

**Comment by SVRKishoreReddy** [comment]
@wimrijnders  Thank you for your reply.
I don't want styling with HTML.
My requirement :For example in the demo, on zooming in,the size of image remains the same but the ellipse(node) and the edge length is increasing. I want only the edge length to increase not the ellipse(node) size and font.

**Comment by wimrijnders** [comment]
Right. This particular functionality is not present and would have to be added, either internally or in the the user code. If I have a bright idea of how to do it, I will mention it here; please also share any ideas you have.

I'm wondering, what would happen in your scenario if you *zoom out*? The nodes would remain the same size, nodes would shorten to oblivion.....I am visualizing a huge conglomerate of nodes now. Worst case would be that only a single node, the topmost one, is visible.

**Comment by SVRKishoreReddy** [comment]
@wimrijnders  if we zoom out,the nodes which are collapsing into each other will form  clusters.

**Comment by wimrijnders** [comment]
Aha! That sounds familiar. I bet you're using [this example](http://visjs.org/examples/network/other/clusteringByZoom.html) as inspiration.

**Comment by SVRKishoreReddy** [comment]
Yes, something like that :)

**Comment by romak1984** [comment]
@SVRKishoreReddy Did you succeeded in getting the desired functionality?
Thanks
